app/routes/index/chapter.js

```diff
@@ -12,49 +12,4 @@ export default Ember.Route.extend({
 
     // return this.store.find('chapter', params.id);    // Use this.store to query the mock database
   },
-  actions: {
-    saveTag(member, chapter, question, option) {
-      // We are passing member, chapter, question here even though we already have it
-      // on the index route. This is to allow the rest of the app to create tags if needed.
-
-      window.console.log("Saving tag locally goes here...");
-
-      this.store.createRecord('tag', {
-        member: member,
-        chapterId: chapter.id,
-        questionId: question.id,
-        answer: option,
-      });
-
-      Ember.$.ajax({
-        method: "POST",
-        url: "/api/v1/responses",
-        data: {
-          memberId: member.id,
-          chapterId: chapter.id,
-          questionId: question.id,
-          answer: option,
-        }
-      });
-    },
-    saveTags(member) {
```

Comment:
Although not a show-stopper, this method should check for tags with empty answers to cut down on spamming the API. Or they empty answer tags should be removed before this is called.